package.json

```diff
@@ -8,6 +8,8 @@
   "scripts": {
     "install": "scripts/install.sh",
     "postinstall": "scripts/postinstall.sh",
+    "preshrinkwrap": "npm cache clean",
+    "postshrinkwrap": "license-reporter save --ignore-version-range --xml licenses.xml && license-reporter report --silent",
```

Comment:
This now makes `npm shrinkwrap` dependent on a devDependency. Previously, because of a limitation on how we used to build our tar archives with `grunt-fh-build`, we needed to tell people to run `npm shrinkwrap` with only production dependencies present. Now though, our minimum target is Node.js 6 which comes with npm 3+, and with that change we had to rethink how we pack our archives, so we no longer have the limitation of needing to run shrinkwrap without devDependencies.

If this change is accepted in all of our components, I'll notify people of the new way.

Reply:
It license-reporter installed in the Jenkins environment? It looks like this is using a globally installed binary of it instead of the one from node_modules?

Reply:
@pb82 When you run things from npm scripts, then `./node_modules/.bin` is added to the start of the PATH, so it's made available from there. It's not included globally in the Jenkins environment.